### ISSUE TITLE
[feature proposal]

### ISSUE THREAD

**Issue by Yupjun** [xmed-lab/CLIP_Surgery#9]
![image](https://user-images.githubusercontent.com/25025997/235296813-cfb19a06-9ed3-408b-9b2a-7d338665e1f4.png)

As you know, tensor activations tend to act noisy on unseen input image. So for the clarity of segmentation, it would be better if you use masking based on tensor values. Please check on Pull requests

**Comment by Eli-YiLi** [comment]
Thanks for your interest on our work. 
Yes, it deepens on users to add a threshold. Besides, torch.clamp is also able to achieve your goal. 
Also, cv2.threshold(similarity_map, 0, 255, cv2.THRESH_BINARY + cv2.THRESH_OTSU) can choose auto threshold for your need.
